### ISSUE TITLE
Last Page Only Footer

### ISSUE THREAD

**Issue by FryDerm** [westy92/html-pdf-chrome#236]
Is there a way to do a footer that only appears on the last page?

**Comment by real-artswan** [comment]
I'm also wondering is it possible to disable header/footer for the first page (making some kind of "cover page")?

**Comment by westy92** [comment]
I don't believe this is possible. We're limited by these underlying options: https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-printToPDF

**Comment by westy92** [comment]
This discussion may help: https://github.com/puppeteer/puppeteer/issues/2089

**Comment by Shivaniben** [comment]
@FryDerm have you find any solution? I am also looking to have last page with different footer. Any help would be appreciated. Thanks!
